settings.js

```diff
@@ -109,9 +108,8 @@ function generate(){
       },
       "filter" : {
         "ampersand" :{
-          "type" : "pattern_replace",
```

Comment:
yea classic case of unit testing tunnel vision. it did what I expected, but also did a bunch more!